inputs/metadata-parse-paired-mgx-mtx.ipynb

```diff
@@ -0,0 +1,5079 @@
+{
```

Comment:
<p>mRNA enriched would be preferable here, total RNA will be largely ribosomal. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/2022-mtx-not-in-mgx-pairs/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>good catch! I actually only used mRNA-enriched per our slack conversation, I'll update this comment to reflect that!</p>